build/templates/UmbracoProject/appsettings.Development.json

```diff
@@ -34,6 +34,9 @@
       },
       //#endif
       "Global": {
+        //#if (HasNoNodesViewPath)
+        "NoNodesViewPath": "NO_NODES_VIEW_PATH_FROM_TEMPLATE",
+        //#endif
```

Comment:
Should this not be the path on cloud (production) too, and therefore be in appsettings.json?

Reply:
Yes please check with @sitereactor if the envs in cloud need this as well

Reply:
I would assume that this should be in appsettings.json given that the setting will be read locally. At least locally is where its most relevant. But applies to all environments.

Reply:
Good point.  As you say, locally is where it's most used, but I guess it could also be needed say to restore from live --> development environment, where both are in Cloud.  I'll amend this.

Reply:
That's done now.